### DISCUSSION TITLE
Ads Site

### DISCUSSION THREAD

**Discussion by Globalfun32y439y54** [UseInterstellar/Interstellar#819] [General]
I was wondering what website do you use for ads?

**Comment by qrs** [comment]
We currently use https://adsterra.com

Although, they are not recommended due to NSFW ads & scam Ads.

**Reply by Globalfun32y439y54** [reply]
Ok Thanks

**Reply by Globalfun32y439y54** [reply]
But how do you add the ads to code?

**Reply by MalcombLee** [reply]
yo whats up

**Reply by MalcombLee** [reply]
Bro check this music out

[https://www.soundtrap.com/studio/_vhN8dwHTROWPsGuKHjCeQ/](https://soundtrap.com/studio/_vhN8dwHTROWPsGuKHjCeQ/ )
